tests/custom_if_prefix/libnetwork_env_var_test.go

```diff
@@ -0,0 +1,66 @@
+package custom_if_prefix
+
+import (
+	"fmt"
+	"math/rand"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	mathutils "github.com/projectcalico/libnetwork-plugin/utils/math"
+	. "github.com/projectcalico/libnetwork-plugin/utils/test"
```

Comment:
Any particular reason this last one is a '.' import?  We generally avoid that except for the gingko/gomega stuff.